easybuild/easyblocks/generic/pythonpackage.py

```diff
@@ -685,6 +685,8 @@ def test_step(self, return_output_ec=False):
 
                 if return_output_ec:
                     (out, ec) = run_cmd(cmd, log_all=False, log_ok=False, simple=False)
+                    # need to log seperately, since log_all and log_ok need to be false to retreive out and ec
+                    self.log.info('cmd "%s" exited with exit code %s and output:\n%s' % (cmd, ec, out))
```

Comment:
Nitpicking: it's better to not use `%` in log statement (see https://medium.com/flowe-ita/logging-should-be-lazy-bc6ac9816906).

And in general, we should outer double quotes for sentences, and "group" things in a message with single quotes:

```suggestion
                    self.log.info("cmd '%s' exited with exit code %s and output:\n%s", cmd, ec, out)
```

Reply:
Done